core/controllers/admin_test.py

```diff
@@ -513,11 +521,11 @@ def test_rollback_exploration_to_safe_state_action(self):
 
         snapshot_content_model = (
             exp_models.ExplorationSnapshotContentModel.get(
-                '0-5', strict=False))
+                '0-5', strict=True))
```

Comment:
Why did this change?

Reply:
I changed False -> True, because keeping False returns Optional type and the return value of this function is never expected to be None while testing. So, to fix the type I have two choices either user assert or True.